js/scripts.js

```diff
@@ -8,15 +8,16 @@
 
     Description: This file contains all the scripts associated with the single-page
     portfolio website.
+
 */
 
-(function($) {
+(function ($) {
```

Comment:
In the future merging linter fixes first would be nice so it doesn't clutter the diff

Reply:
Look a conversation!

Reply:
It would be nice, but that really isn't a problem going from develop to master, only from a feature branch to develop.